peerprep-backend/src/main/java/com/peerprep/peerprepbackend/entity/User.java

```diff
@@ -0,0 +1,29 @@
+package com.peerprep.peerprepbackend.entity;
+
+import jakarta.persistence.*;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+/**
+ * Entity class representing a user profile row in PostgreSQL
+ * To be stored in the "users" table
+ */
+
+@Data
+@Builder
+@Entity
+@AllArgsConstructor
+@NoArgsConstructor
+@Table(name = "users")
+public class User {
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private Long id;
+
+    private String username;
+    private String email;
+    private String country;
+    private String password;
```

Comment:
These can be annotated with `@NotNull` for automatic enforcement (in the app level, it doesn't enforce anything about the actual db columns), add spring's validation starter dependency to use it

https://www.baeldung.com/java-validation

Reply:
ok added, thanks!!